Command.md

```diff
@@ -0,0 +1,309 @@
+<!-- omit in toc -->
+# Developer guide
+
+This guide helps you get started developing kubi
+
+Make sure you have the following dependencies installed before setting up your developer environment:
+
+ - Git
+ - Jq
+ - Go
+ - In this context we will use a kind cluster for the local deployment of Kubi : cluster kind {1.23 or 124}
+
+## Deploy kubi  
+
+ - Get the source code https://github.com/ca-gip/kubi
+  
+ - install go  https://go.dev/
+    
+ - Create kind cluster with version 1.24-1.26 https://kind.sigs.k8s.io/docs/user/quick-start/
+  
+ - kubi private key
+ ```
+ openssl ecparam -genkey -name secp521r1 -noout -out /tmp/ecdsa-key.pem
```

Comment:
/tmp/ should not be used, except if you are using mktemp.

Please adapt instructions. 

Can we make this in a single script ?